backend/grants/admin.py

```diff
@@ -238,6 +240,27 @@ def send_grant_reminder_to_waiting_for_confirmation(modeladmin, request, queryse
         messages.info(request, f"Grant reminder sent to {grant.name}")
 
 
+@admin.action(description="Send Waiting List update email")
+def send_reply_email_waiting_list_update(modeladmin, request, queryset):
+    queryset = queryset.filter(
+        status__in=(
+            Grant.Status.waiting_list,
+            Grant.Status.waiting_list_maybe,
+        ),
+    )
+
+    for grant in queryset:
+        if grant.applicant_reply_sent_at is not None:
```

Comment:
uhm is this check correct? I imagine we already sent an email so this field will not be None for everyone